### ISSUE TITLE
New mod Vehicle Monitor inspired by VehicleExplorer ?

### ISSUE THREAD

**Issue by Lactic68** [sperrgebiet/FS22_VehicleExplorer#13]
Hi @sperrgebiet,
Did you see that a new mod [FS22 Vehicle Monitor](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=225552&title=fs2022) has been published today by Modhub which seems to offer same functionalities as your mod ?

**Comment by sperrgebiet** [comment]
Well, there are a couple of such mods out there. VehicleMonitor looks more like a VehicleInspector clone. And sure, also VeEx got inspired by those, at least the 'Vehicle Status' part in the info boxes. I personally see it as an advantage within the modding community. I know that a lot often complain about 'stolen ideas'. Well, if someone takes the idea of VeEx, makes it better than it is, then awesome. I started with this because I missed that functionality. I just don't like it when somebody would take my work and pretends it is theirs. But as long as there are some kudos mentioned, why not. After all we all can just learn from each other and become better by that. And if you look into my code, there are some parts which I took from others as well, but usually I add a note.

So basically this was now a long version of: "Thanks for letting me know, but actually I don't care" ;)
